app/models/cms/page.rb

```diff
@@ -91,25 +91,24 @@ def mime_type
   def to_xml(options = {})
     xml = options[:builder] || Nokogiri::XML::Builder.new
 
-    xml.page do |x|
+    xml.__send__(CMS::XML::CLASS_TAG.call(self.class)) do |x|
       unless new_record?
         x.id id
         x.created_at created_at.xmlschema
         x.updated_at updated_at.xmlschema
       end
-
       x.title title
+      x.section_id section_id
+      x.path path
+      x.layout_id layout_id
       x.system_name system_name
-      x.path(path) if respond_to?(:path)
-      x.hidden hidden?
-      x.layout layout_name
```

Comment:
Also I don't remember discussing the removal of this one.

Reply:
Same, removed for being fixed, empty or not applicable fields for this type.